src/lib/tvgPictureImpl.h

```diff
@@ -96,8 +98,15 @@ struct Picture::Impl
                 auto scene = loader->scene();
                 if (scene) {
                     paint = scene.release();
-                    resizing = true;
                     loader->close();
+
+                    if (w == 0 && h == 0) {
+                        w = loader->w;
+                        h = loader->h;
+                    } else {
+                        resize();
```

Comment:
I think this call(resize) is unnecessary.
The resize function is called in update by the resizing flag.

Reply:
it needs. 
size update()-> resize() -> render() -> reload()